MBBSEmu/HostProcess/MbbsHost.cs

```diff
@@ -119,18 +132,39 @@ public MbbsHost(ILogger logger, IGlobalCache globalCache, IFileUtility fileUtili
             _incomingSessions = new Queue<SessionBase>();
             _cleanupTime = ParseCleanupTime();
             _timer = new Timer(unused => _performCleanup = true, this, NowUntil(_cleanupTime), TimeSpan.FromDays(1));
-
+            _nightlyCleanupTimer = new Timer(ProcessNightlyCleanup, this, TimeSpan.FromSeconds(1), TimeSpan.FromSeconds(1));
```

Comment:
Is this really needed? I would think running the cleanup routine in the original WorkerThread would be better since it eliminates many of the multithreading issues?

Or is the point to let WorkerThread die so you can create a new one? Though I'm not sure how that's possible since WorkerThread is guarded against `!= null` but is never set back to `null`

Reply:
This was added to "spin" the worker routine to let the cleanup run without RTICK etc. running , in testing this was the solve to prevent anything from going on while cleanup is running and to not run it again if currently running

```
            _nightlyCleanupTimer = new Timer(ProcessNightlyCleanup, this, TimeSpan.FromSeconds(1), TimeSpan.FromSeconds(1));
```